sass/_navigation.scss

```diff
@@ -0,0 +1,97 @@
+.bg__color--blue {
+	background-color: $blueish;
+
+	.body__container--nav {
+		max-width: 1300px;
+		margin: 0 auto;
+		display: flex;
+		justify-content: space-between;
+		align-items: flex-end;
+		padding: 10px 40px;
+
+		@media screen and (max-width: $largeMobile) {
+			padding: 10px 15px;
+		}
+
+		.nav__name {
+			font-size: 1.5rem;
+			color: $white;
+		}
+
+		.nav__burger {
+			display: none;
+
+			@media screen and (max-width: $largeMobile) {
+				display: block;
+			}
+		}
+		.nav__items {
+			position: absolute;
+			top: 70px;
+			right: 15px;
+			display: flex;
+			flex-direction: column;
+			justify-content: space-evenly;
+			width: 345px;
+			height: 300px;
+			text-align: center;
+			background-color: $white;
+			z-index: 10;
+
+			&::before {
+				content: '';
+				position: absolute;
+				top: -20px;
+				right: 0;
+				border-bottom: 30px solid #fff;
+				border-left: 30px solid transparent;
+			}
+
+			.nav__item {
+				margin-left: 50px;
+				color: $white;
+
+				&:hover {
+					color: $darkGrey;
+				}
+
+				&:nth-last-child(1) {
+					border-radius: 30px;
+					padding: 10px 20px;
+					font-family: $fontSerif;
+					color: $darkGrey;
+					background-color: $white;
+					text-transform: uppercase;
+
+					&:hover {
+						color: $white;
+						background-color: rgb(255, 255, 255, 0.2);
+					}
+				}
+
+				@media screen and (max-width: $largeMobile) {
+					color: $mediumGrey;
+					margin-left: 90px;
+					margin-right: 90px;
+
+					&:nth-last-child(1) {
+						background-color: $yellow;
+					}
+				}
+			}
+		}
+	}
+}
+
+@media screen and (max-width: $largeMobile) {
+	.bg__color--blue {
+		.body__container--nav {
+			.show {
```

Comment:
@Jelena-ag czy nie powinno tu być inne nazewnictwo? Jeśli coś robi "display: none", to raczej bym się spodziewała, że taka klasa będzie się nazywała np. _hidden_, a nie _show_

Reply:
w sumie racja, zmienię to